models.py

```diff
@@ -0,0 +1,184 @@
+        db_table = "db_movie_actors"
+        unique_together = (("movie", "actor"),)
+
+
+class DbMovieGenres(models.Model):
+    movie = models.ForeignKey(DbMovie, models.DO_NOTHING)
+    genre = models.ForeignKey(DbGenre, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = "db_movie_genres"
+        unique_together = (("movie", "genre"),)
+
+
+class DbMoviesession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey(DbCinemahall, models.DO_NOTHING)
+    movie = models.ForeignKey(DbMovie, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = "db_moviesession"
+
+
+class DbOrder(models.Model):
+    created_at = models.DateTimeField()
+    user = models.ForeignKey("DbUser", models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = "db_order"
+
+
+class DbTicket(models.Model):
+    row = models.IntegerField()
+    seat = models.IntegerField()
+    movie_session = models.ForeignKey(DbMoviesession, models.DO_NOTHING)
+    order = models.ForeignKey(DbOrder, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = "db_ticket"
+        unique_together = (("row", "seat", "movie_session"),)
+
+
+class DbUser(models.Model):
+    password = models.CharField(max_length=128)
+    last_login = models.DateTimeField(blank=True, null=True)
+    is_superuser = models.BooleanField()
+    username = models.CharField(unique=True, max_length=150)
+    first_name = models.CharField(max_length=150)
+    last_name = models.CharField(max_length=150)
+    email = models.CharField(max_length=254)
+    is_staff = models.BooleanField()
+    is_active = models.BooleanField()
+    date_joined = models.DateTimeField()
+
+    class Meta:
+        managed = False
+        db_table = "db_user"
+
+
+class DbUserGroups(models.Model):
+    user = models.ForeignKey(DbUser, models.DO_NOTHING)
+    group = models.ForeignKey(AuthGroup, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = "db_user_groups"
+        unique_together = (("user", "group"),)
+
+
+class DbUserUserPermissions(models.Model):
+    user = models.ForeignKey(DbUser, models.DO_NOTHING)
+    permission = models.ForeignKey(AuthPermission, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = "db_user_user_permissions"
+        unique_together = (("user", "permission"),)
+
+
+class DjangoContentType(models.Model):
+    app_label = models.CharField(max_length=100)
+    model = models.CharField(max_length=100)
+
+    class Meta:
+        managed = False
+        db_table = "django_content_type"
+        unique_together = (("app_label", "model"),)
+
+
+class DjangoMigrations(models.Model):
+    app = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
+    applied = models.DateTimeField()
+
+    class Meta:
+        managed = False
+        db_table = "django_migrations"
```

Comment:
There is no need for this file, create a regular migration in the `db` app to encounter all model changes and leave the default structure of the django DB initialization.